modules/asset-manager-workflowoperation/src/main/java/org/opencastproject/workflow/handler/assetmanager/AssetManagerStorageMoveOperationHandler.java

```diff
@@ -67,19 +67,25 @@ public WorkflowOperationResult start(WorkflowInstance workflowInstance, JobConte
     //A missing version is ok, that just means to select all of them (which is represented as null)
     String targetVersion = StringUtils.trimToNull(operation.getConfiguration("target-version"));
     Version version = null;
-    if (null != targetVersion) {
+    if (null != targetVersion && !"*".equals(targetVersion)) {
```

Comment:
Thanks for reviewing:)
this flag was added, because we needed the use case that all versions of  mediapackage can be moved at once. According to the documentation, the `target-version` should simply be omitted and all versions of the mediapackage are moved to the cloud storage, but an error occurred while trying out. The reason was the lack of a `target-version` in the workflow operation. Therefore, the flag was introduced to indicate that all versions of a media package should be moved to the cloud storage.

Reply:
Whoops, that's a mistake on my part.  I'll have a PR out shortly, and then you should be able to drop the `*` flag.